packages/performance-profiler-plugin/package.json

```diff
@@ -26,7 +26,7 @@
   "scripts": {
     "prebuild": "npm run clean",
     "build": "webpack --mode=development",
-    "build:prod": "npm run build",
+    "build:prod": "cross-env NODE_ENV=production webpack",
```

Comment:
I think I removed `cross-env` and instead used the preferred `webpack --mode=production`. Does that work the same. If so, we should use that since it's the standard webpack approach.

Reply:
Yeah I can change that.